worlds/dark_souls_3/docs/setup_en.md

```diff
@@ -25,27 +25,26 @@ To downpatch DS3 for use with Archipelago, use the following instructions from t
 
 1. Launch Steam (in online mode).
 2. Press the Windows Key + R. This will open the Run window.
-3. Open the Steam console by typing the following string: steam://open/console , Steam should now open in Console Mode.
-4. Insert the string of the depot you wish to download. For the AP supported v1.15, you will want to use: download_depot 374320 374321 4471176929659548333.
+3. Open the Steam console by typing the following string: `steam://open/console` , Steam should now open in Console Mode.
```

Comment:
You can remove this space and this should be a semicolon or period instead of a comma.
```suggestion
3. Open the Steam console by typing the following string: `steam://open/console`. Steam should now open in Console Mode.
```